wikimedia/wikipedia-ios
# mobile html progress

WIP on mobile html

Testing steps:

1.  Fresh install
2. Tap featured article
3.  Confirm it loads. Tap link. Confirm it pushes.
4. Tap bookmark icon at bottom
5. Go back to Explore, enter airplane mode. Tap featured article again. This is pulling from URLCache.
6. Go to settings, clear cache, go back to Explore. Tap featured article again. This is pulling from Core Data.

Delete steps:
1. Tap featured article
2. Tap bookmark icon
3. Enter airplane mode. Go to Settings and clear cache.
4. Confirm tapping featured article still loads article.
5. Toggle off bookmark icon, go back to Explore
6. Confirm tapping featured article again no longer loads.
